content/actions/learn-github-actions/security-hardening-for-github-actions.md

```diff
@@ -25,7 +25,7 @@ Secrets use [Libsodium sealed boxes](https://libsodium.gitbook.io/doc/public-key
 
 To help prevent accidental disclosure, {% data variables.product.product_name %} uses a mechanism that attempts to redact any secrets that appear in run logs. This redaction looks for exact matches of any configured secrets, as well as common encodings of the values, such as Base64. However, because there are multiple ways a secret value can be transformed, this redaction is not guaranteed. As a result, there are certain proactive steps and good practices you should follow to help ensure secrets are redacted, and to limit other risks associated with secrets:
 
-- **Never use structured data as a secret**
+- **Never use unstructured data as a secret**
```

Comment:
@brentjo This change makes sense to me, I wanted to also get your 👀 just to make sure 🙂

Reply:
👋 @lucasilverentand thanks for opening this up, and for catching this. `Structured` was the intended word here though, so the typo's actually with how we go on to say `Unstructured` in the first sentence. As in: don't use "structured" data such as a blob of JSON as a secret, because secret redaction mostly looks for exact matches to the configured secrets, so doing so can cause redaction to fail. You want to configure the secret in its rawest form that you would want redacted in the event that value ever gets written to the logs. 

"Structured" was the best term I could think of that captures that meaning, but definitely open to suggestions on that!

Reply:
@brentjo that makes a lot of sense. I do agree with you that the term makes sense. Is there any chance someone else could make the change? Since I'm currently a bit busy with other projects.

Reply:
@lucasilverentand Can do! Thanks again for bringing this to our attention!

Reply:
@brentjo glad I could help! 😇